Sources/FoundationEssentials/FileManager/FileManager+Utilities.swift

```diff
@@ -193,15 +193,15 @@ extension _FileManagerImpl {
             #else
             var result: Int32
             if followSymLinks {
-                result = lsetxattr(path, key, buffer.baseAddress!, buffer.count, 0)
-            } else {
                 result = setxattr(path, key, buffer.baseAddress!, buffer.count, 0)
+            } else {
+                result = lsetxattr(path, key, buffer.baseAddress!, buffer.count, 0)
```

Comment:
Is there a unit test that could be added to validate this behavior to confirm that the change works as intended and it doesn't regress in the future? The `FileManager` test suite has some scaffolding to support creating a test directory with symlinks and calling file manager operations on files

Reply:
Yeah I'll add it